### PR TITLE
Build branch Code review

### PR DESCRIPTION
**Pull request by sanadwj**
### This project is a simple Todo List webpage where one can be able to do the following:

- [x] Dynamically rendering a simple todo homepage.
- [x] View all projects
- [x] View all todos in each project.
- [x] Expand a single todo to see/edit its details
- [x] Delete a todo

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @sanadwj  👋

**You've done pretty well on this. 👌**

However, let's make a few changes.

- [ ] Please check comments on your code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
-        with:
-          node-version: "12.x"
-      - name: Setup Stylelint
-        run: |
-          npm install --save-dev stylelint@13.3.x stylelint-scss@3.17.x stylelint-config-standard@20.0.x stylelint-csstree-validator
-          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/javascript/.stylelintrc.json
-      - name: Stylelint Report
+name: Linters
+
+on: [push,pull_request]
```
Comment (mikenath223) [inline .github/workflows/linters.yml]: - [ ] This should be `on: pull_request`. Please make the change.

File: src/forms/todoForm.js [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+    document.removeEventListener('keydown', deleteForm2);
+  }
+
+  function showWarning(element) {
+    element.style.display = 'block';
+
+    setTimeout(() => {
+      element.style.display = 'none';
+    }, 6000);
+  }
```
Comment (mikenath223) [inline src/forms/todoForm.js]: - [ ] Also kindly change these to ES6 arrow function syntax. Please make the change in other affected functions too.

**Comment by sanadwj** [comment]
@mikenath223 , Thank you for the review. I fixed the required changes

**Review by trekab — APPROVED** [review]
### Approved! :tada:
Well done, @sanadwj! You may proceed to your next project.

 _Happy coding!_ :wink: 
__Treasure Kabareebe__
[Portfolio](https://trekab.github.io/portfolio/) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Pull request merged**